modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -26,13 +26,17 @@ The guided decision tables feature works similarly to ...
 .Example 2. Procedure Module
 [source]
 ----
-[#creating-guided-decision-tables]
+[id='creating-guided-decision-tables']
 = Creating Guided Decision Tables
 
 You can use guided decision tables to ...
 ----
 
-NOTE: The format defined here is recommended because it supports attributes that enable topics to be reused and cross-referenced properly. For details, see xref:reusing-modules[]. You can also use the format `+++[id='anchor-name']+++`, but be consistent in your choice of anchor formatting.
+[NOTE]
+.Note on Other Anchor Formats (Not Recommended)
```

Comment:
Unrelated to asciidoc style specifically, but we've usually not altered the default heading of an admonition because it created work for localization (default DocBook headings were already translated in the spec). [This](https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/language-support.adoc) seems to suggest that the same is not true for asciidoc so this should be fine, but worth verifying.

Reply:
Wow @laubai, way to dig deep on that one! :) I personally think the value that custom block headings add in certain cases justifies the work they would require for localization. I do avoid them most of the time, but they're hard to resist on occasions where they optimize scannability and usability so much. Excellent food for thought, though.